templates/imdb/templates.yaml

```diff
@@ -0,0 +1,56 @@
+dataset: imdb
+templates:
+  02ff2949-0f45-4d97-941e-6fa4c0afbc2d: !Template
+    id: 02ff2949-0f45-4d97-941e-6fa4c0afbc2d
+    jinja: The following movie review expresses what sentiment? {{text}} ||| {{ ["negative",
+      "positive"] [label] }}
+    name: imdb_5
+    reference: ''
+  866474a5-1498-46b7-bfee-ac0c5160707f: !Template
+    id: 866474a5-1498-46b7-bfee-ac0c5160707f
+    jinja: '{{text}}How does the viewer feel about the movie? ||| {{ ["negative",
+      "positive"] [label] }}'
+    name: imdb_6
+    reference: ''
+  96538f30-f2c1-430e-8fc6-936a16966d9c: !Template
+    id: 96538f30-f2c1-430e-8fc6-936a16966d9c
+    jinja: '{{text}} What sentiment does the writer express for the movie? ||| {{
+      ["negative", "positive"] [label] }}'
+    name: imdb_7
+    reference: ''
+  af51297c-38a3-4d6c-a8b5-04b1243d7443: !Template
+    id: af51297c-38a3-4d6c-a8b5-04b1243d7443
+    jinja: '{{text}}The sentiment expressed for the movie is ||| {{ ["negative", "positive"]
+      [label] }}'
+    name: imdb_3
+    reference: ''
+  b93b74ac-fe95-40b4-9610-318b46ab820f: !Template
+    id: b93b74ac-fe95-40b4-9610-318b46ab820f
+    jinja: '''{{text}} What is the sentiment expressed in this text? ||| {{ ["negative",
+      "positive"] [label] }}'''
+    name: imdb_1
+    reference: ''
+  bd82ba0f-01d4-4fa1-bf8d-07e392c00cd9: !Template
+    id: bd82ba0f-01d4-4fa1-bf8d-07e392c00cd9
+    jinja: '{{text}} Did the reviewer enjoy the movie? ||| {{ ["Yes", "No"] [label]
+      }}'
+    name: imdb_9
+    reference: ''
+  c70d1687-2421-49a2-9553-91b8bac4cfbe: !Template
+    id: c70d1687-2421-49a2-9553-91b8bac4cfbe
+    jinja: '{{text}} What is the sentiment expressed by the reviewer for the movie?
+      ||| {{ ["negative", "positive"] [label] }}'
+    name: imdb_4
+    reference: ''
+  e01970ab-42c0-4e6e-a08f-4940d889ef37: !Template
+    id: e01970ab-42c0-4e6e-a08f-4940d889ef37
+    jinja: '{{text}}How does the reviewer feel about the movie? ||| {{ ["They loved
+      it!", "They din''t like it"] [label] }}'
```

Comment:
Labels are backwards.
`din''t` -> `didn't`

Reply:
I think this is how its stored in the templates file. The prompt is correct as seen here -
![image](https://user-images.githubusercontent.com/9425893/120933591-67a7c200-c6c0-11eb-9f6a-1d0d1c36924b.png)

Reply:
I think it's rendering ok. There are a few issues though. By "labels backwards" I mean positive is mapped to "they didn't like it." Didn't is misspelled. And there should be a space between `{{text}}` and the question.

Reply:
Ah. Ok. Fixed it. Chat lingo habits - dint is more easy to write that didnt :).